pkg/skaffold/jib/jib_darwin.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2018 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package jib
+
+func getWrapper() string {
+	return "gradlew"
+}
+
+func getCommand(workspace string, defaultExecutable string, defaultSubCommand []string) (executable string, subCommand []string) {
```

Comment:
Can you capitalize this so that it can be used elsewhere?

Or provide `InvokeMaven` and `InvokeGradle` variants that hide the wrapper bit.

Reply:
I think I'll leave that to another PR - one that would use this method from another package - it's probably good to keep visibility as low as possible until it needs to be exposed.

Reply:
I want to replace what I have :-)

Reply:
Would making `getCommandMaven` and `getCommandGradle` exposed be good? I'll change it to include a custom subcommand to use.